lib/index.js

```diff
@@ -77,20 +77,24 @@ var buildPackage = function(obj, callback) {
       }
     });
 
-    var manifestFile = fse.createOutputStream(path.join(destination, 'imsmanifest.xml'));
-    manifestFile.write(manifest(schemaVersion, obj));
-    _logSuccess(path.join(destination, 'imsmanifest.xml'));
+     manifestFile.write(manifest(schemaVersion, obj), function(err) {
+      if (err) _logError(err);
```

Comment:
The _logError is simple console.log with some formatting. So please return after you log error as we have to stop it here. You can also throw error here I don't mind.